litecord/blueprints/guild/members.py

```diff
@@ -40,6 +42,17 @@ async def get_guild_member(guild_id, member_id):
     member = await app.storage.get_single_member(guild_id, member_id)
     return jsonify(member)
 
+@bp.route("/<int:guild_id>/members/<int:member_id>", methods=["PUT"])
+async def add_guild_member(guild_id, member_id):
+    """Forcibly add a member to a guild"""
+    await admin_check()
+
+    # TODO: if we ever support bots we will need to use checks for all of these here but 
+    # for now since this is only for the admin panel we can safely skip all checks
+    await add_member(guild_id, member_id, skip_check=True, basic=True)
```

Comment:
Setting basic to True here skips adding the `@everyone` role and stuff as well as a few other crucial things such as dispatching a `GUILD_MEMBER_ADD`.

Reply:
Understood, but what I'm worried about is that `basic=False` checks for invites disabled, which administrators really should be able to bypass

Reply:
As I mentioned above, `add_member()` should probably be modified to use `skip_check` as the toggle for the feature checks instead of `basic` so `basic` can be set to False. Make sure other things using it don't break.